src/IndexBuilder.php

```diff
@@ -74,8 +72,10 @@ public function speedUpRefresh(Index $index): void
 
     public function migrate(Index $current, Index $new)
     {
-        // @todo Waiting for https://github.com/ruflin/Elastica/pull/1637 to be merged
-        // This method should use the TASK API, because we do not want to WAIT for the reindex (HTTP Timeout issues).
+        $reindex = new Reindex($current, $new);
+        $reindex->setWaitForCompletion(Reindex::WAIT_FOR_COMPLETION_FALSE);
+
+        return $reindex->run();
```

Comment:
As stated in the original comment, I would like this method to wait (or allow to wait).
But we can't wait on the original HTTP call for timeout reasons. So we should use the task ID and the Task API to monitor this long running process.

Could you put back the comment maybe?